x-pack/plugins/task_manager/server/metrics/task_run_metrics_aggregator.ts

```diff
@@ -6,14 +6,26 @@
  */
 
 import { JsonObject } from '@kbn/utility-types';
-import { isOk, unwrap } from '../lib/result_type';
+import { merge } from 'lodash';
+import { isOk, Ok, unwrap } from '../lib/result_type';
 import { TaskLifecycleEvent } from '../polling_lifecycle';
-import { ErroredTask, RanTask, TaskRun } from '../task_events';
+import {
+  ErroredTask,
+  RanTask,
+  TaskRun,
+  isTaskManagerStatEvent,
+  isTaskRunEvent,
+  TaskManagerStat,
+} from '../task_events';
 import { MetricCounterService } from './counter/metric_counter_service';
+import { SerializedHistogram, SimpleHistogram } from './simple_histogram';
 import { ITaskMetricsAggregator } from './types';
 
 const taskTypeGrouping = new Set<string>(['alerting:', 'actions:']);
 
+const HDR_HISTOGRAM_MAX = 1800; // 30 minutes
+const HDR_HISTOGRAM_BUCKET_SIZE = 10; // 10 seconds
```

Comment:
@kobelb I set the bucket size to 10 seconds as per the SLI doc suggestion, maxing out at 30 minutes (this was based on a recent SDH where I believe we saw task delays in this range). WDYT?

Reply:
Sounds good to me.